packages/math/src/schnorr.cairo

```diff
@@ -0,0 +1,127 @@
+//! bip340 implementation
```

Comment:
Rename the module back to bip-340, it is not a general purpose schnorr, it uses double sha256 as a hash function which is very bitcoin specific.

Reply:
Done!